pkg/discovery/kubernetes/kubernetes.go

```diff
@@ -177,24 +177,6 @@ func (c *ContainerDefinition) Labels() []*profilestorepb.Label {
 	}}
 }
 
-func (c *ContainerDefinition) PerfEventCgroupPath() string {
```

Comment:
Hehe, was going to delete this soon, glad you are removing it here! Makes me quite happy to see this gone 😄